tasks/metalsmith.js

```diff
@@ -190,12 +190,15 @@ function build (buildCount) {
       done()
     })
     .use(function (files, metalsmith, done) {
-        // hack to add stats for AMF
+      // hack to add stats for AMF
       var meta = metalsmith.metadata()
       meta.stats.amfCostPerLifeSaved = 2838
-      // meta.stats.numberMembers = meta.stats.TotalPledges
-      // meta.stats.amountDonated = meta.stats.TotalPayments
-      // meta.stats.amountPledged = meta.stats.TotalLiability
+      meta.stats.numberMembers = meta.stats.TotalGWWCMembers
+      // hardcode numbers from database
+      // parfit=> SELECT sum(amount_normalized)::MONEY from pledges.reported_donation;
```

Comment:
What's all the dead code?

Reply:
This is just to leave a clue how to get this number until we add completion payments and do it properly